src/commands/apphosting-secrets-set.ts

```diff
@@ -90,12 +91,14 @@ export const command = new Command("apphosting:secrets:set <secretName>")
 
     // Note: The API proposal suggested that we would check if the env exists. This is stupidly hard because the YAML may not exist yet.
     let path = config.yamlPath(process.cwd());
-    let yaml: config.Config = {};
+    let projectYaml: yaml.Document;
```

Comment:
As this method gets longer, I would recommend breaking it out into helper functions to help users more easily parse through the top level control flow, as we do here: https://github.com/firebase/firebase-tools/blob/master/src/apphosting/index.ts#L99